deploy/helm/sumologic/templates/otelcol-deployment.yaml

```diff
@@ -42,12 +42,17 @@ spec:
         resources:
           {{- toYaml .Values.otelcol.deployment.resources | nindent 10 }}
         ports:
-        - containerPort: 55678 # Default endpoint for Opencensus receiver.
+        - containerPort: 5778  # Default endpoint for Jaeger Sampling.
+        - containerPort: 6831  # Default endpoint for Jaeger Thrift Compact receiver.
+          protocol: UDP
+        - containerPort: 6832  # Default endpoint for Jaeger Thrift Binary receiver.
+          protocol: UDP
+        - containerPort: 8888  # Default endpoint for querying metrics.
```

Comment:
It would be nice to add it to the prometheus :)

Reply:
I can take care about it, some day :D

Reply:
This is a very good point, we have been asked about that already BTW